### PR TITLE
VACMS-2096: EWA contrib module collaboration setup & rules of engagement

### PR DESCRIPTION
**Pull request by oksana-c**
## Description

See #2096  

* minimal structure for `workflow_assignments` module to support collaboration
* `READMES/ewa-rules-of-engagement.md` provides the outline of EWA contribution process
* `docroot/module/contrib/workflow_assignments` included in phpcs checks. PHPCS will review this directory on every commit. This directive will eventually go away once we fully move to DO.

## QA steps

- [ ] EWA contribution process is described in https://github.com/department-of-veterans-affairs/va.gov-cms/pull/2330/files#diff-49bfebaf74f385f43d140d3c47d4e4aa. Feedback is welcome
- [ ] pull this branch locally and run `lando composer install`
- [ ] verify that `workflow_assignments` module is pulled into `docroot/module/contrib` directory
- [ ] `cd docroot/module/contrib/workflow_assignments`, then `git status` and confirm you're on `1.x` branch of EWA

### CODE REVIEW

**Comment by oksana-c** [comment]
need a better exclusion statement for theme .css for vagovadmin theme

File: composer.json [hunk 1]
```diff
@@ -378,7 +383,7 @@
             "find docroot/modules/custom docroot/themes -name '*.install' -print0 | xargs -0 -n1 php -l > /dev/null"
         ],
         "va:test:cs": [
-            "phpcs --ignore=*.md,*.min.css,styles.css,wysiwyg.css,proofing.css,*/node_modules/*,*/simplesaml*/* --extensions=php,module,inc,install,profile,engine,theme,js,css --standard=./docroot/vendor/drupal/coder/coder_sniffer/Drupal/ruleset.xml docroot/modules/custom docroot/themes/custom --colors"
```
Comment (swirtSJW) [inline composer.json:381]: I think proofing.css got clobbered during a conflict.

Comment (swirtSJW) [reply composer.json:381]: Nevermind... I see you added a whole path that should cover it.

Comment (oksana-c) [reply composer.json:381]: see commit a58392c above your comment ^

**Pull request merged**